operators/revaha/__init__.py

```diff
@@ -0,0 +1,172 @@
+import hashlib
+import re
+
+import dataflows as DF
+import requests
+import slugify
+
+from conf import settings
+from srm_tools.logger import logger
+from srm_tools.processors import ensure_fields, update_mapper
+from srm_tools.update_table import airflow_table_updater
+
+
+def transform_phone_numbers(r):
+    phone_numbers = r['authority_phone'] if r['authority_phone'] else ''
+    machlaka_phone = r['machlaka_phone'] if r['machlaka_phone'] else ''
+    if machlaka_phone:
+        phone_numbers += machlaka_phone
```

Comment:
q: are these strings? how are they separated? is it guaranteed that just one of them would have a value?